deploy/kubeturbo-for-openshift/README.md

```diff
@@ -236,9 +223,10 @@ spec:
 ##### Deploy Kubeturbo Pod
 
 ```console
-$kubectl create -d kubeturbo-openshift.yaml
+$oc create -d kubeturbo-openshift.yaml
```

Comment:
"-d" is not a valid flag. should be "-f":
oc create -f kubeturbo-openshift.yaml